docs/core/tools/dotnet-install-script.md

```diff
@@ -92,7 +92,7 @@ You can install a specific version using the `-Version|--version` argument. The
 
 - **`-InstallDir|--install-dir <DIRECTORY>`**
 
-  Specifies the installation path. The directory is created if it doesn't exist. The default value is *%LocalAppData%\Microsoft\dotnet*. Binaries are placed directly in this directory.
+  Specifies the installation path. The directory is created if it doesn't exist. The default value is *%LocalAppData%\Microsoft\dotnet* on Windows and */usr/share/dotnet* on Linux. Binaries are placed directly in this directory.
```

Comment:
Is there a common variable we can use in this like `LocalAppData` is for Windows? I think that your linux path is actually a macos path. My linux machine is dying and I need to refresh to double check.

Reply:
https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet#environment-variables doesn't mention such a variable:

> The default location on Linux and macOS is `/usr/share/dotnet`.

Reply:
yes. I'm unsure why I had a different path. I had asked someone back before I got a mac to check with.